packages/synapse-react-client/src/components/DataGrid/SynapseGrid.tsx

```diff
@@ -112,13 +112,19 @@ const SynapseGrid = forwardRef<
     const { columnNames, columnOrder, rows } = modelSnapshot
     const gridRows = rows.map(row => {
       const rowObj: { [key: string]: any } = {}
```

Comment:
I think we could improve the type/structure of `DataGridRow`; for now let's just use it here so it is clear that we need to fix it here if/when we update it
```suggestion
    const gridRows = rows.map((row): DataGridRow => {
      const rowObj: DataGridRow = {}
```


To clarify what I mean, a `DataGridRow` should probably explicitly declare that it has validation information. So it could look something like

```ts
type DataGridRow = {
  data: Record<string, string | number | boolean | null | undefined>
  validationInformation?: ValidationResults
}

```